integration_tests/test_commit_format.py

```diff
@@ -3,14 +3,13 @@
 """Test the commit message format."""
 
 import os
-import subprocess
 
-from utils import get_cmd_output
+from utils import get_cmd_result
 
 COMMIT_TITLE_MAX_LEN = 50
 COMMIT_BODY_LINE_MAX_LEN = 72
-BASE_BRANCH = os.environ['BUILDKITE_PULL_REQUEST_BASE_BRANCH']
-BASE_REPO = os.environ['BUILDKITE_REPO']
+BASE_BRANCH = os.environ.get('BUILDKITE_PULL_REQUEST_BASE_BRANCH', 'master')
+BASE_REPO = os.environ.get('BUILDKITE_REPO', 'origin')
```

Comment:
I would document a bit the default values, including when this test can be run locally, something like: `The commit format test works properly on the local machine only when the targeted branch of the pull request is "master" and the remote name of the upstream repository is "origin". If the targeted branch is for example a feature branch, then this test won't work as expected. In the rust-vmm CI of each repository these limitations no longer exist.`

Reply:
Done.

Reply:
Can we also rename this to `BASE_REMOTE` or something else closer to what we actually use it for?

Reply:
Done.